catho/quantum
# Fix/pagination responsive

## What this PR does:
Fix the `pagination` component that breaks the responsibility when you changes the browser size from desktop to mobile.

## How to test
1. Create a new react project and call the pagination component from quantum. Or run a project that have this component.
2. Build quantum project (with this branch) with `npm run build` and get the built code and paste inside the node modules folder in the project used in 1. (the path is `./node_modules/@catho/quantum`.
3. Run the project 1 and go to the `pagination` component.
4. Resize the page from desktop to mobile and check if it breaks.
